composer.json

```diff
@@ -15,11 +15,11 @@
     ],
     "require": {
         "php": ">=5.3.3",
-        "league/commonmark": "dev-master"
```

Comment:
I disagree. The tests are already failing due to breaking changes from 0.5->0.7...

We need to keep the minor version locked.